operations/app/terraform/vars/staging/locals.tf

```diff
@@ -2,15 +2,18 @@ locals {
   environment = "staging"
   address_id  = 6
   init = {
-    environment           = local.environment
-    location              = "eastus"
-    is_metabase_env       = true
-    resource_group_name   = "prime-data-hub-${local.environment}"
-    resource_prefix       = "pdh${local.environment}"
-    okta_redirect_url     = "https://${local.environment}.prime.cdc.gov/download"
-    okta_base_url         = "hhs-prime.oktapreview.com"
-    OKTA_scope            = "simple_report_dev"
-    storage_queue_name    = ["process", "batch", "batch-poison", "elr-fhir-convert", "process-poison", "send", "send-poison", "elr-fhir-convert", "elr-fhir-convert-poison", "elr-fhir-route", "elr-fhir-translate", "elr-fhir-translate-poison", "process-elr"]
+    environment          = local.environment
+    location             = "eastus"
+    is_metabase_env      = true
+    resource_group_name  = "prime-data-hub-${local.environment}"
+    resource_prefix      = "pdh${local.environment}"
+    okta_redirect_url    = "https://${local.environment}.prime.cdc.gov/download"
+    okta_base_url        = "hhs-prime.oktapreview.com"
+    OKTA_scope           = "simple_report_dev"
+    RS_okta_redirect_url = "https://prime-data-hub-XXXXXXX.azurefd.net/download"
+    RS_okta_base_url     = "reportstream.oktapreview.com"
+    RS_OKTA_scope        = "simple_report_dev"
```

Comment:
`RS_OKTA_scope` is `"reportstream_dev"`

Reply:
RS_OKTA_scope has been updated in the latest commit